File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 1 - C/q6.c
//Faça um algoritmo que leia a base e a altura e imprima o valor do perimetro, da area e da diagonal.
#include<stdio.h>
int main(){
	float base, altura, perim, area, diagonal;
	printf("Insira a base: ");
	scanf("%f", &base);
	printf("Insira a altura: ");
	scanf("%f", &altura);
	perim=2*(base+altura);
	area=base*altura;
	diagonal=sqrt(base*base+altura*altura);
	printf("Perimetro: %f\n", perim);
	printf("Area: %f\n" , area);
	printf("Diagonal: %f\n", diagonal);		
}

File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 1 - Python/q6.py
#Faça um algoritmo que leia a base e a altura e imprima
#o valor do perimetro, da area e da diagonal.

import math
base=float(input("Insira a base do retângulo: "))
altura=float(input("Insira a altura do retângulo: "))
perim=2*(base+altura)
area=base*altura
diagonal=math.sqrt(base**2+altura**2)
print("Perimetro: ", perim)
print("Area: " , area)
print("Diagonal: ", diagonal)
